src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -1824,7 +1824,7 @@ End Module</Document>)
         End Function
 
         <WorkItem(546208, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/546208")>
-        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        <WpfFact(Skip:="https://github.com/dotnet/roslyn/issues/11980"), Trait(Traits.Feature, Traits.Features.Completion)>
```

Comment:
I will merge with this change in master before submitting.

---
In reply to: [67015552](https://github.com/dotnet/roslyn/pull/11981#discussion_r67015552) [](ancestors = 67015552)